test/cli/samples/basic/actual/Main.js

```diff
@@ -0,0 +1,173 @@
+/* src/Main.html generated by Svelte v2.3.0 */
```

Comment:
Do we want to do whatever string replacement we do in other tests here, so that this comment won't have to change each version?

Reply:
good idea — have done that